lib/rgrb/plugin/keyword/generator.rb

```diff
@@ -12,7 +12,8 @@ class Generator
         include PluginBase::Generator
 
         # cre.jp 検索ページの URL
-        CRE_SEARCH_URL = 'http://cre.jp/search/?sw=%s'
+        # %s が与えられたキーワードを URL エンコードした物に置換される
```

Comment:
コメントをより簡潔な文章にしてみました。

```suggestion
        # %s には、URL エンコードされたキーワードが入る
```

Reply:
修正しました。